bin/edbg-env.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Is this edbg-env.sh file used by anything else other than the edbgBmcWrapper?  If not, this file doesn't need to exist.  The one line export could go into edbgBmcWrapper.sh for now and then that line removed at a later date.

Reply:
If this is used by something else - it should reference bmc in the name as this is not a generic edbg environment setup script.

Reply:
Yes. you are right. I initially did that only but it didn't work initially. then, only I have added this another script and did source <script> to set the env for current session